src/components/Block/Actions.vue

```diff
@@ -1,15 +1,17 @@
 <template>
   <Block v-if="plugins.length > 0 && ts >= payload.end" :title="$t('actions')">
-    <UiButton
-      v-for="plugin in plugins"
-      :key="plugin"
-      @click="execute(plugin)"
-      :loading="loading"
-      :disabled="!$auth.isAuthenticated.value"
-      class="width-full button--submit"
-    >
-      {{ $t('submitOnchain') }}
-    </UiButton>
+    <div v-if="loaded">
```

Comment:
There should be a loading if it's not loaded, you can test it here `/#/dai/proposal/QmdNR3Eed8nmNPMciQkTznQfR98sDcXVbLpGqUZ996M8tk`